sep490-frontend/src/app/modules/enterprise/components/users/users.component.ts

```
@@ -88,11 +107,75 @@
+      });
+  }
+
+  // eslint-disable-next-line max-lines-per-function
+  private deleteUsers(): void {
+    this.loading = true;
+    const userIds = this.selected.map(user => user.id);
+
+    this.userService
+      .deleteUsers(userIds)
+      .pipe(
+        finalize(() => (this.loading = false)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)